extensions/packageManager/package.json

```diff
@@ -55,6 +55,22 @@
     "@types/node": "^14.6.2",
     "@types/react": "^16.9.53",
     "rimraf": "^3.0.2",
+    "@typescript-eslint/eslint-plugin": "2.34.0",
+    "@typescript-eslint/parser": "2.34.0",
+    "eslint": "7.0.0",
+    "eslint-config-prettier": "6.11.0",
+    "eslint-plugin-cypress": "2.11.1",
+    "eslint-plugin-emotion": "10.0.27",
+    "eslint-plugin-format-message": "6.2.3",
+    "eslint-plugin-import": "2.20.2",
+    "eslint-plugin-jsx-a11y": "6.2.3",
+    "eslint-plugin-lodash": "7.1.0",
+    "eslint-plugin-notice": "0.9.10",
+    "eslint-plugin-prettier": "3.1.3",
+    "eslint-plugin-react": "7.20.0",
+    "eslint-plugin-react-hooks": "4.0.2",
+    "eslint-plugin-security": "1.4.0",
+    "prettier": "2.0.5",
```

Comment:
There is already a prettier config in the extensions root. You can just use vs code to format without having to add all of these dependencies.

Reply:
OK removed!